sonar-project.properties

```diff
@@ -2,6 +2,5 @@ sonar.organization=sonarsource
 sonar.projectKey=SonarSource_gh-action_release
 sonar.python.version=3.10
 sonar.python.coverage.reportPaths=main/build/coverage.xml
-sonar.sources=main
-sonar.exclusions=main/tests/*
+sonar.sources=main/release
```

Comment:
What is the rationale of this?

Reply:
No, it's fine. I just didn't understand at first. Thanks for the clarification 👍